configs/default/BEFH-BETAFPVF411RX.config

```diff
@@ -0,0 +1,108 @@
+# Betaflight / STM32F411 (S411) 4.1.0 Jun 17 2019 / 23:11:43 (f7f0c6780) MSP API: 1.42
+
+board_name BETAFPVF411RX
+manufacturer_id BEFH
+
+# resources
+resource MOTOR 1 B08
+resource MOTOR 2 B01
+resource MOTOR 3 B10
+resource MOTOR 4 B07
+resource PPM 1 A10
+resource LED 1 C14
+resource LED_STRIP 1 A00
+resource ADC_BATT 1 A01
+resource ADC_CURR 1 B00
+resource SERIAL_TX 1 A09
+resource SERIAL_RX 1 A10
+resource SERIAL_TX 2 A02
+resource SERIAL_RX 2 A03
+resource SERIAL_TX 11 A08
+resource SERIAL_RX 11 A08
+
+# GYRO & ACC
+resource GYRO_CS 1 A04
+resource SPI_SCK 1 A05
+resource SPI_MISO 1 A06
+resource SPI_MOSI 1 A07
+resource GYRO_EXTI 1 B06
+
+# OSD
+resource OSD_CS 1 B12
+resource SPI_SCK 2 B13
+resource SPI_MISO 2 B14
+resource SPI_MOSI 2 B15
+
+# SPI RX
+resource RX_SPI_CS 1 A15
+resource SPI_SCK 3 B03
+resource SPI_MISO 3 B04
+resource SPI_MOSI 3 B05
+resource RX_SPI_EXTI 1 C13
+resource RX_SPI_BIND 1 B02
+resource RX_SPI_LED 1 C15
+resource RX_SPI_CC2500_TX_EN 1 B09
+resource RX_SPI_CC2500_LNA_EN 1 A13
+resource RX_SPI_CC2500_ANT_SEL 1 A14
+
+timer A00 AF1
+# pin A00: TIM2 CH1 (AF1)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer A10 AF1
+# pin A10: TIM1 CH3 (AF1)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer B08 AF2
+# pin B08: TIM4 CH3 (AF2)
+timer B10 AF1
+# pin B10: TIM2 CH3 (AF1)
+
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin A00 0
+# pin A00: DMA2 Stream 1 Channel 6
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin B10 0
+# pin B10: DMA1 Stream 1 Channel 3
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SPI
+feature OSD
+feature SOFTSERIAL
+feature TELEMETRY
+feature OSD
+feature AIRMODE
+
+# serial
+
+serial 30 2048 115200 57600 0 115200
+
+# master
+set rx_spi_protocol = FRSKY_X
+set rx_spi_bus = 3
+set rx_spi_led_inversion = ON
+set dshot_burst = AUTO
+set dshot_idle_value = 450
+set dshot_burst = AUTO
+set dshot_bitbang = OFF
+set motor_pwm_protocol = DSHOT600
```

Comment:
We combine the board with 4 in 1 ESCS.I think the default DSHOT600 is a good choice.

Reply:
If this setting is used as a default, and if your board is used in combination with older ESCs that do not support Dshot, this will result in the motors spinning up as soon as a battery is connected, potentially resulting in damage or injury.
Do you assume all responsibility for personal injury and damage resulting from using this default setting?

Reply:
OK,I understand, I submitted the amendment again and deleted the DSHOT protocol as an alternative to the PWM protocol.Thanks.